db/dcrpg/internal/chartblockstmts.go

```diff
@@ -0,0 +1,25 @@
+package internal
+
+const (
+	// InsertChartBlock holds DML for Chart block data inserting.
+	InsertChartBlock = `INSERT INTO chartblocks VALUES ($1, $2, $3);`
+
+	// SelectChartBlockByTimeRangeSQL holds the query string for selecting chart data by a time range.
+	SelectChartBlockByTimeRangeSQL = `SELECT * FROM chartblocks WHERE time BETWEEN $1 AND $2 ORDER BY time LIMIT $3;`
+
+	// SelectAllChartBlocks holds the query string for selecting chart data.
+	SelectAllChartBlocks = `SELECT * FROM chartblocks ORDER BY time;`
+
+	// CreateChartBlockTable holds DDL string for creating the chart data table.
+	CreateChartBlockTable = `CREATE TABLE IF NOT EXISTS chartblocks (
```

Comment:
I still don't get why we need a table for this.  For two reasons it's a problem: (1) The data is already in existing tables, and (2) this functionality won't work in lite mode.